### PR TITLE
Add templates path for ALBA Synchrotron

### PR DESCRIPTION
**Pull request by jordiandreu**
Add PathTemplate model for ALBA synchrotron

### CODE REVIEW

File: HardwareObjects/queue_model_objects_v1.py [hunk 1]
```diff
@@ -1312,6 +1312,27 @@ def get_archive_directory(self):
                                              PathTemplate.archive_folder)
             archive_directory = os.path.join(archive_directory,
                                              *folders[4:])
+        elif PathTemplate.synchotron_name == "ALBA":
```
Comment (beteva) [inline HardwareObjects/queue_model_objects_v1.py]: It is unfortunate that we have to add a path for each institute. I think all these paths (ESRF, ALBA, ...) should go to Session.

File: HardwareObjects/queue_model_objects_v1.py [hunk 2]
```diff
@@ -1327,6 +1327,27 @@ def get_archive_directory(self):
                                              PathTemplate.archive_folder)
             archive_directory = os.path.join(archive_directory,
                                              *folders[4:])
+        elif PathTemplate.synchotron_name == "ALBA": 
+            #archive_directory = os.path.join(PathTemplate.archive_base_directory)
```
Comment (IvarsKarpics) [inline HardwareObjects/queue_model_objects_v1.py]: I would suggest to remove not used code (#...). I agree with @beteva  that this should go to Session hwobj. We can discuss this during the developers meeting.

File: HardwareObjects/queue_model_objects_v1.py (hunk 1 above)
Comment (jordiandreu) [reply HardwareObjects/queue_model_objects_v1.py]: Indeed.

File: HardwareObjects/queue_model_objects_v1.py (hunk 2 above)
Comment (jordiandreu) [reply HardwareObjects/queue_model_objects_v1.py]: Done!

**Comment by IvarsKarpics** [comment]
If it is fine for everyone I will accept the PR and if we will find time during the developers meeting we can discuss on how to move this code to Session.py

**Comment by marcus-oscarsson** [comment]
Yes, I think we all agree on this and we will consider it at some stage in the refactoring work

**Pull request merged**